corehq/motech/repeaters/tasks.py

```diff
@@ -256,17 +260,29 @@ def iter_ready_repeater_ids_forever():
                 continue
             if not domain_can_forward_now(repeater.domain):
                 continue
-            yielded = True
-            yield repeater.domain, repeater.repeater_id
+
+            lock = get_repeater_lock(repeater.repeater_id)
+            lock_token = uuid.uuid1().hex  # The same way Lock does it
```

Comment:
> Why use `uuid1` here rather than the more standard `uuid4`?

This is what the comment refers to. Sorry it wasn't more clear. I'll make it clearer.

I set the value of `lock_token` [the same way that `redis.lock.Lock` sets its lock token](https://github.com/redis/redis-py/blob/master/redis/lock.py#L198).

> Possibly add a comment to explain why it is needed?

Will do. (Redis uses the lock token to determine the ownership of the lock. If a process knows the value of the token, then that process is considered the owner of the lock, and it is allowed to release the lock without first acquiring it.)

Reply:
d1119bbec47